progress/react/icons/src/index.ts

```diff
@@ -2,3 +2,4 @@ export { Battery } from './Battery';
 export { Heart } from './Heart';
 export { Pie } from './Pie';
 export { Ups } from './Ups';
+export { ProgressIcon } from './ProgressIcon';
```

Comment:
Does this need to be exported from the index?

Reply:
You're right this doesn't need to be exposed for end-users.